ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/shacl/entities/ShaclShape.java

```diff
@@ -0,0 +1,37 @@
+package be.vlaanderen.informatievlaanderen.ldes.server.domain.shacl.entities;
+
+import org.apache.jena.rdf.model.Model;
+
+import java.util.Objects;
+
+public class ShaclShape {
+	private final String collection;
+	private final Model model;
+
+	public ShaclShape(String collection, Model model) {
+		this.collection = collection;
+		this.model = model;
+	}
+
+	public String getCollection() {
+		return collection;
+	}
+
+	public Model getModel() {
+		return model;
+	}
+
+	@Override
+	public boolean equals(Object o) {
```

Comment:
can you add unit tests for the equals and hashcode? You can use a parameterized test for this. Cfr. ViewNameTest for a complete parameterized example or TokenTest for a hybrid example.